src/app.py

```diff
@@ -323,8 +354,38 @@ def load_file(filename: str) -> dict:
     st.subheader(":blue-background[Secret Scanning Alerts]")
     st.write("Alerts open for more than 5 days.")
 
+    col1, col2 = st.columns(2)
+    with col1:
+        start_date_slo = st.date_input(
+            "Start Date", datetime.now().date() - pd.DateOffset(years=1), key="start_date_slo"
+        )
+    with col2:
+        end_date_slo = st.date_input("End Date", datetime.now().date(), key="end_date_slo")
+
+    if end_date_slo < start_date_slo:
+        st.error("End date cannot be before start date.")
+        st.stop()
+
+    # Filter the secret scanning alerts by the selected date range
+    df_secret_scanning["created_at"] = pd.to_datetime(df_secret_scanning["created_at"], errors="coerce").dt.tz_localize(
+        None
+    )
+    df_secret_scanning = df_secret_scanning.dropna(subset=["created_at"])
```

Comment:
Same as above:

is .dropna() needed? shouldn't all records have a created_at value?